gui/qt/main_window.py

```diff
@@ -1428,6 +1447,9 @@ def do_send(self, preview = False):
         if (fee < (tx.estimated_size())):
             msg.append(_('Warning') + ': ' + _("You're using a fee less than 1000 sats/kb.  It may take a very long time to confirm."))
 
+        if ((self.message_opreturn_e.text() is not None) and (self.message_opreturn_e.text() !="")):
+            msg.append(_('Warning') + ': ' + _("You are using an OP_RETURN message.  This gets written PERMANENTLY in the blockchain!"))
```

Comment:
This warning sounds a bit forceful and evil.  You could not use all caps for PERMANENTLY since... I don't like to get yelled at by my programs.

Get rid of "Warning" and make PERMANENTLY be lowercase is my suggestion.